test/integration/029_docs_generate_tests/test_docs_generate.py

```diff
@@ -1381,7 +1373,6 @@ def expected_postgres_references_manifest(self, model_database=None):
             },
             'metadata': {
                 'project_id': '098f6bcd4621d373cade4e832627b4f6',
-                'user_id': None,
```

Comment:
Another field that feels like it should be included even when it's None

Reply:
Agreed! Let's discuss in the thread about `loaded_at_field`